### PR TITLE
Pressing ENTER (blank string) triggers the hint

### PR DESCRIPTION
**Pull request by johnpaulashenfelter**
Keyboard users rejoice! Pressing ENTER on the keyboard (with an empty string) triggers a hint just like pressing the Hint button. Enjoy!

Resolves: #46

### CODE REVIEW

File: src/components/card.js [hunk 1]
```diff
@@ -16,7 +16,11 @@ class Card extends Component {
     event.preventDefault();
     const guess = this._guess.value;
     this._guess.value = "";
-    this.props.guess(guess);
+    if(guess === '') {
+      this.handleHintClick(event)
```
Comment (jasonaowen) [inline src/components/card.js]: I missed it during pairing, but now that I look at it on GitHub I notice that we missed the trailing semicolon here. In this case it's unambiguous, and we don't yet have a linter hooked up, but would you mind adding one?

Comment (johnpaulashenfelter) [reply src/components/card.js]: Done

**Comment by johnpaulashenfelter** [comment]
@jasonaowen Squash before merge?

**Comment by jasonaowen** [comment]
> Squash before merge?

Yes, please!

**Pull request merged**